frontend/device/devicelistener.lua

```diff
@@ -150,43 +207,29 @@ function ReaderFrontLight:onShowOnOff()
     return true
 end
 
-function ReaderFrontLight:onShowIntensity()
-    if not Device:hasFrontlight() then return true end
-    local powerd = Device:getPowerDevice()
+function DeviceListener:onShowFlDialog()
+    local FrontLightWidget = require("ui/widget/frontlightwidget")
+    UIManager:show(FrontLightWidget:new{
+        use_system_fl = Device:hasLightLevelFallback()
+    })
+end
```

Comment:
It does keep the diff nice and clean but before merging it's probably better to indent for future readers.

Reply:
done